dali/python/nvidia/dali/pipeline.py

```diff
@@ -497,24 +527,25 @@ def __exit__(self, exception_type, exception_value, traceback):
         """Safely restores previous pipeline."""
         Pipeline.pop_current()
 
-
     def add_sink(self, edge):
-        """Allows to manual add of graph edges to the pipeline which are not connected to the output and all pruned
+        """Allows to manual add of graph edges to the pipeline which are not connected to
+        the output and all pruned
```

Comment:
```suggestion
```suggestion
        """Marks an edge as a data sink, preventing it from being pruned, even if it's not
        connected to the pipeline output.
```
```